README.md

```diff
@@ -29,28 +25,28 @@
 
 ## Documentation
 
-### Material UI
-
-Visit [https://mui.com/material-ui/](https://mui.com/material-ui/) to view the full documentation.
+Get started in the [Material UI documentation](https://mui.com/material-ui/getting-started/).
 
 <details>
   <summary>Older versions</summary>
 
-- **[v4.x](https://v4.mui.com/)** ([Migration from v4 to v5](https://mui.com/material-ui/migration/migration-v4/))
-- **[v3.x](https://v3.mui.com/)** ([Migration from v3 to v4](https://mui.com/material-ui/migration/migration-v3/))
-- **[v0.x](https://v0.mui.com/)** ([Migration to v1](https://mui.com/material-ui/migration/migration-v0x/))
+- **[v5.x](https://v5.mui.com/)** ([Upgrading from v5 to v6](https://mui.com/material-ui/migration/upgrade-to-v6/))
+- **[v4.x](https://v4.mui.com/)** ([Upgrading from v4 to v5](https://mui.com/material-ui/migration/migration-v4/))
+- **[v3.x](https://v3.mui.com/)** ([Upgrading from v3 to v4](https://mui.com/material-ui/migration/migration-v3/))
+- **[v0.x](https://v0.mui.com/)** ([Upgrading to v1](https://mui.com/material-ui/migration/migration-v0x/))
 
 </details>
 
-**Note:** `@next` only points to pre-releases.
+**Note:** `@next` points to pre-releases.
 Use `@latest` for the latest stable release.
 
-### Joy UI
+## Joy UI
+
+This repository also contains Joy UI, an experimental component library that implements our own in-house Joy Design. Joy UI is in beta and _development is currently on hold_. When starting a new project from scratch, we recommend Material UI over Joy UI because we can guarantee ongoing support.
```

Comment:
Line breaks?

```suggestion
This repository also contains Joy UI, an experimental component library that implements our own in-house Joy Design.
Joy UI is in beta and _development is currently on hold_.
When starting a new project from scratch, we recommend Material UI over Joy UI because we can guarantee ongoing support.
```